docker-compose.yml

```diff
@@ -37,6 +42,20 @@ services:
       - network.host=0.0.0.0
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+      - ELASTICSEARCH_URL=https://elasticsearch:9200
+      - ELASTICSEARCH_IP=https://elasticsearch
+      - ELASTICSEARCH_PORT=9200
+      - SETUP_PASSWORDS=yes
```

Comment:
A better name would be something like:
`XPACK_SECURITY_ENABLED`

Reply:
Every related variable should start with `XPACK_SECURITY_`.
Example:
`XPACK_SECURITY_ELASTIC_PASSWORD`

Reply:
Changed